packages/kbn-i18n/BUILD.bazel

```diff
@@ -102,38 +102,16 @@ ts_project(
   allow_js = True,
   declaration = False,
   incremental = True,
-  out_dir = "web",
+  out_dir = "target_web",
```

Comment:
There are several places where we ignore the `target` dir, since this `target_web` dir only exists in the bazel output I assume we don't have to worry about that here?

Reply:
that folder will only exist inside the Bazel bin folder, we don't need to worry about it :)